src/python/pants/engine/native.py

```diff
@@ -283,6 +283,20 @@ def extern_get_type_for(self, context_handle, val):
     type_id = c.to_id(type(obj))
     return TypeId(type_id)
 
+  @_extern_decl('Handle', ['ExternContext*', 'TypeId'])
+  def extern_get_handle_from_type_id(self, context_handle, ty):
+    c = self._ffi.from_handle(context_handle)
+    obj = c.from_id(ty.tup_0)
+    return c.to_value(obj)
+
+  @_extern_decl('bool', ['ExternContext*', 'TypeId'])
+  def extern_is_union(self, context_handle, type_id):
+    "Return whether or not a type is a member of a union"
```

Comment:
Convention is to use a triple quoted string for comments.

Reply:
Yes, in fact that is what populates the `__doc__` dunder method. If you use triple quotes immediately after the `def` line, then it becomes `docstring`.

Reply:
Looks like Python 3 doesn't care syntactically whether a docstring is triple-quoted or not, but the docs say that one should use triple-quotes for consistency. I'll change it to triple quotes, and maybe this requirement should be part of the linter?

Reply:
It is, but :(:
https://github.com/pantsbuild/pants/blob/96c152a3ad0bf2421c2fe6ce7295dc07f18b84d3/pants.ini#L284-L303